src/main/java/com/segment/analytics/android/integrations/adobeanalytics/VideoAnalytics.java

```diff
@@ -315,11 +315,13 @@ private void trackVideoContentStarted(TrackPayload track) {
 
   private void trackVideoContentCompleted() {
     trackAdobeEvent(MediaHeartbeat.Event.ChapterComplete, null, null);
-    heartbeat.trackComplete();
-    logger.verbose("heartbeat.trackComplete();");
   }
 
+  //Upon playback complete, pause playhead, call trackComplete, and end session
   private void trackVideoPlaybackCompleted() {
+    playback.pausePlayhead();
+    heartbeat.trackComplete();
+    logger.verbose("heartbeat.trackComplete();");
```

Comment:
Would it also help to log `playback.pausePlayhead();`?

Reply:
If not, this thing looks good to go!

Reply:
I think we're good not logging this since we don't log it on other events like Video Playback Interrupted, Video Playback Paused, Buffer Started, etc!